### DISCUSSION TITLE
Karabiner not working at all

### DISCUSSION THREAD

**Discussion by chenhuiapp** [pqrs-org/Karabiner-Elements#3847] [Q&A]
I am not sure why this happened. I was trying the following settings.
https://github.com/mxstbr/karabiner
but I dont really like it.  After I switched back to my old settings, everything stopped working.

I tried to uninstall and reinstall karabiner, and made a fresh new key modification mapping. But nothing works. Karabiner is using its own default settings to control my keyboard. Is there a way to fix this?

**Comment by lankysam** [comment]
Just thought I'd add that even the space bar is not working in certain apps, e.g. Quicken.  Urgent help needed, thank you
